stable/kured/templates/daemonset.yaml

```diff
@@ -58,6 +58,9 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+          ports:
+            - name: http
+              containerPort: {{ .Values.service.port }}
```

Comment:
Should the service.port and the containerPort always be the same thing? Is there a way to tell kured to run on a different port?